hazelcast/src/main/java/com/hazelcast/sql/impl/client/SqlClientUtils.java

```diff
@@ -38,4 +44,34 @@ public static SqlError exceptionToClientError(Exception exception, UUID localMem
             sqlException.getOriginatingMemberId()
         );
     }
+
+    public static byte expectedResultTypeToByte(SqlExpectedResultType expectedResultType) {
```

Comment:
Consider, moving `expectedResultTypeToByte` and `expectedResultTypeToEnum` method into `SqlExpectedResultType` enum with name something like `toByte` or `expectedResultType` and `fromByte`. 
Then we could use `SqlExpectedResultType#toByte` and would need to change this switch statement.

WDYT?

Reply:
`SqlExpectedResultType` is a public API, I would avoid exposing internals from there.